.github/workflows/multimodal-pr.yml

```
@@ -107,3 +101,41 @@ jobs:
           ipython kernel install --name "python3" --user
           echo "Current branch: ${BRANCH_NAME}"
           papermill --parameters repo_branch ${BRANCH_NAME} notebooks/phrase_grounding.ipynb /tmp/phrase_grounding_output.ipynb
+
+  publish:
+    runs-on: ubuntu-18.04
+    needs: [ flake8, mypy, pytest ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: Install pypa/build
+        run: pip install build
+
+      - name: Build a binary wheel and a source tarball
+        run: python -m build --sdist --wheel --outdir ${{ env.folder }}/dist/ ${{ env.folder }}
+
+      - name: Publish package to TestPyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_TEST_API_TOKEN }}
+          packages_dir: ${{ env.folder }}/dist/
+          repository_url: https://test.pypi.org/legacy/
+          verbose: true
+          skip_existing: true
+
+      - name: Publish package to PyPI
+        if: startsWith(github.ref, 'refs/tags/v')
```

@fepegar, Here's the condition we have in the other publishing pipeline:
```suggestion
        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
```

---

Why is this necessary? Also, could you please check https://github.com/microsoft/hi-ml/pull/472#discussion_r921393130?